### PR TITLE
slight editorial tweaks to Connection

### LINKED ISSUE #1009: Editorial tweaks to Connection
The Connection header field is oddly inconsistent with the other field definitions in how it defines the grammar late in the description. I think it is worth an editorial fix.

### PR DESCRIPTION
**Pull request by royfielding**
Fixes #1009

### CODE REVIEW

**Review by mnot — APPROVED** [review]

File: draft-ietf-httpbis-semantics-latest.xml [hunk 1]
```diff
@@ -2949,7 +2956,7 @@ Host: www.example.org
    connection-option in this field, remove any header or trailer field(s) from
    the message with the same name as the connection-option, and then
    remove the Connection header field itself (or replace it with the
-   intermediary's own connection options for the forwarded message).
+   intermediary's own control options for the forwarded message).
```
Comment (reschke) [inline draft-ietf-httpbis-semantics-latest.xml:2959]: @royfielding - was that change intentional?

Comment (royfielding) [reply draft-ietf-httpbis-semantics-latest.xml:2959]: Yes, they are referred to as control options in the definition. A connection-option is the ABNF term.
